src/initial-parameters.ts

```diff
@@ -37,7 +38,7 @@ export async function determineInitialParameters(
         )
       : path.resolve(cwd, inputs.tempDirectory);
   const parsedTodayTimestamp =
-    TODAY === undefined ? NaN : new Date(TODAY).getTime();
+    TODAY === undefined ? NaN : parseDateAsISO(TODAY).getTime();
```

Comment:
Here `TODAY` is an ISO string. We could pass it straight to the Date constructor, but that would result in a Date in UTC rather than local time. We can work with this Date object using `.getUTCFullYear()`, `.getUTCMonth()`, and `.getUTCDate()`, but I'd rather not place that burden on those of us who will maintain this. To fix this I'm using `parseISO` from `date-fns`, which will give us back a time in local time (e.g. `parseDateAsISO('2022-08-25')` => `new Date(2022, 7, 25)`). I guess ultimately it doesn't matter since we use `formatDate` (also from `date-fns`) later on to turn this back into some kind of date string, but when debugging this it seems more straightforward to see a local time here.

Reply:
Outdated due to changes, resolving.